packages/@aws-cdk/core/package.json

```diff
@@ -37,6 +37,7 @@
     "exclude": [
       "props-physical-name:@aws-cdk/aws-cloudformation.CustomResourceProps",
       "construct-ctor:@aws-cdk/core.App.<initializer>",
+      "construct-ctor:@aws-cdk/core.Construct.<initializer>.params[0]",
```

Comment:
Shouldn't we apply the new rule here?

Reply:
Not yet. Not all of core is migrated yet. 

Trying to do so causes failures downstream that takes me down a rabbit hole. Trying to avoid that. This might have to be the last module to migrate over.